src/components/CallView/CallView.vue

```diff
@@ -426,12 +428,16 @@ export default {
 			this.adjustSimulcastQuality()
 		},
 
-		speakers() {
-			this._setPromotedParticipant()
+		speakers(value) {
+			if (value) {
+				this._setPromotedParticipant()
+			}
```

Comment:
What is value here?

Reply:
Value is the speakers array. If it is empty, it won't update the `promoted` attribute ( if we call it here, we will have no promoted participant because of the clearance inside `_setPromotedParticipant()`)